### DISCUSSION TITLE
Mumbai chain is gone

### DISCUSSION THREAD

**Discussion by KoYoungWock** [ProjectOpenSea/seaport-js#526] [Ideas]
I heard that the Mumbai chain will disappear now, so I think it would be good if you could provide the introduction of the Amoy chain

So I Forked that repo to fly PR, but only some codes are Forked, is there any way?

https://blog.thirdweb.com/goodbye-mumbai-hello-amoy/
https://polygon.technology/blog/introducing-the-amoy-testnet-for-polygon-pos

**Comment by hx2ryu** [comment]
I heart that too. Do you have a plan to support that? 😭

**Comment by ryanio** [comment]
Yes we are adding support for Amoy on OpenSea to replace Mumbai in the coming week or two

**Reply by hx2ryu** [reply]
@ryanio Sounds great! Thank you for your answer.

**Reply by KoYoungWock** [reply]
@ryanio Thank you !!!!!!

**Reply by ryanio** [reply]
Amoy is now live on OpenSea: https://docs.opensea.io/changelog/polygon-amoy-testnet

**Comment by ryanio** [comment]
Amoy is now live on OpenSea: https://docs.opensea.io/changelog/polygon-amoy-testnet

**Reply by KoYoungWock** [reply]
I'm using 'Seaport Library' right now.
So we need 'Seaport Library' that supports Amoy at the moment.

So, is the work supporting Amoy going on in seaport js library???
Actually I want that support to seaport js
